agent/consul/leader_test.go

```diff
@@ -335,6 +335,7 @@ func TestLeader_CheckServersMeta(t *testing.T) {
 
 	retry.Run(t, func(r *retry.R) {
 		member.Tags["nonvoter"] = "1"
```

Comment:
Our future selves would probably appreciate a deprecated comment here.
```suggestion
		// DEPRECATED - remove nonvoter tag in favor of read_replica in a future version of consul
		member.Tags["nonvoter"] = "1"
```